test/source/tests/setup.ts

```diff
@@ -430,6 +431,16 @@ AN8G3r5Htj8olot+jm9mIa5XLXWzMNUZgg==
         { isSavePassphraseChecked: false, isSavePassphraseHidden: false });
     }));
 
+    ava.default('setup - enterprise users should be redirected to their help desk when an error occured', testWithBrowser(undefined, async (t, browser) => {
+      if (testVariant === 'ENTERPRISE-MOCK') {
+        const acctEmail = 'flowcrypt.compatibility@gmail.com';
+        const settingsPage = await browser.newPage(t, TestUrls.extensionSettings());
+        const oauthPopup = await browser.newPageTriggeredBy(t, () => settingsPage.waitAndClick('@action-connect-to-gmail'));
+        await OauthPageRecipe.mock(t, oauthPopup, acctEmail, 'login_with_invalid_state');
+        settingsPage.waitForContent('@container-error-modal-text', 'If this happens again, please contact your Help Desk');
```

Comment:
You can have if/else only on this statement. For enterprise, test for this sentence, for consumer, test the other one. Plus, missing an `await`.

Reply:
Yes, sir... Also, I finished updating each sentence dynamically to what is appropriate. Thank you!